tests/callbacks_test.go

```diff
@@ -246,3 +247,63 @@ func (s *FunctionalSuite) TestWorkflowNexusCallbacks_CarriedOverContinueAsNew()
 		}
 	}
 }
+
+func (s *FunctionalSuite) TestWorkflowCallbacks_ForceActivityComplete() {
+	dc := s.testCluster.host.dcClient
+	dc.OverrideValue(s.T(), dynamicconfig.FrontendEnableCallbackAttachment, true)
+	defer dc.RemoveOverride(dynamicconfig.FrontendEnableCallbackAttachment)
+
+	sdkClient, err := client.Dial(client.Options{
+		HostPort:  s.testCluster.GetHost().FrontendGRPCAddress(),
+		Namespace: s.namespace,
+	})
+	s.NoError(err)
+
+	activityErrCh := make(chan error, 1)
+	mockErrorActivity := func(ctx context.Context) error {
+		err = errors.New("mock error")
+		activityErrCh <- err
+		return err
+	}
+	wf := func(ctx workflow.Context) error {
+		ao := workflow.ActivityOptions{
+			StartToCloseTimeout: 3 * time.Minute,
+			RetryPolicy: &temporal.RetryPolicy{
+				InitialInterval: 2 * time.Minute,
+			},
+		}
+		ctx2 := workflow.WithActivityOptions(ctx, ao)
+		var err error
+		workflow.ExecuteActivity(ctx2, mockErrorActivity).Get(ctx2, &err)
+		return err
+	}
+
+	taskQueue := s.randomizeStr(s.T().Name())
+	workflowType := "test"
+	s.NoError(err)
+
+	w := worker.New(sdkClient, taskQueue, worker.Options{})
+	w.RegisterWorkflowWithOptions(wf, workflow.RegisterOptions{Name: workflowType})
+	w.RegisterActivity(mockErrorActivity)
+	s.NoError(w.Start())
+	defer w.Stop()
+
+	ctx := NewContext()
+	workflowOptions := client.StartWorkflowOptions{
+		ID:        uuid.New(),
+		TaskQueue: taskQueue,
+	}
+	_, err = sdkClient.ExecuteWorkflow(ctx, workflowOptions, wf)
+	s.NoError(err)
+ 	<-activityErrCh
```

Comment:
It's better to poll on the describe response. The activity attempt failure is not guaranteed to have made it to the service here.

Reply:
Would you mind to elaborate on the polling here? 
What dose the "describe response" reference to here?

Reply:
```go
s.EventuallyWithT(func(t *assert.CollectT) {
	desc, err := sdkClient.DescribeWorkflowExecution(ctx, run.GetID(), run.GetRunID())
	require.NoError(t, err)
	assert.Equal(t, 1, len(desc.PendingActivities))
	assert.NotNil(t, desc.PendingActivities[0].LastFailure)
})
```